### PR TITLE
DOC-4055-Delta-Sync

### PR DESCRIPTION
**Pull request by ibsoln**
https://issues.couchbase.com/browse/DOC-4055

### CODE REVIEW

**Comment by ibsoln** [comment]
Need to also include changes to Learn section and CLI per the ticket

File: modules/ROOT/pages/getting-started.adoc [hunk 1]
```diff
@@ -493,6 +493,10 @@ Configuration properties:
 <3> The xref:shared-bucket-access.adoc[shared bucket access] feature allows Couchbase Server SDKs to also perform operations on this bucket.
 <4> `num_index_replicas` is the number of index replicas stored in Couchbase Server, introduced with xref:indexing.adoc[GSI/N1QL indexing].
 If you're running a single Couchbase Server node for development purposes the `num_index_replicas` must be set to `0`.
+<5> *Enterprise Edition*: By default, the `delta_sync.enabled` property is disabled on Sync Gateway.
+When enabled, Couchbase Lite clients will switch to using delta sync automatically.
+Similarly, if delta sync is disabled on Sync Gateway, clients will switch to normal mode.
+This feature is describe in xref:config-properties.adoc#databases-foo_db-delta_sync[databases.$db.delta_sync.enabled].
```
Comment (jamesnocentini) [inline modules/ROOT/pages/getting-started.adoc]:
```suggestion
This feature is described in xref:config-properties.adoc#databases-foo_db-delta_sync[databases.$db.delta_sync.enabled].
```

**Review by jamesnocentini — COMMENTED** [review]
Testing the review functionality

**Review by RichardSmedley — APPROVED** [review]
LGTM

**Pull request merged**